spinalcordtoolbox/scripts/sct_register_to_template.py

```diff
@@ -1192,7 +1192,7 @@ def register(src, dest, step, param):
         )
 
     elif step.algo == 'slicereg':
-        warp_forward_out, warp_inverse_out = register_step_ants_slice_regularized_registration(
+        warp_forward_out, warp_inverse_out, _ = register_step_ants_slice_regularized_registration(
```

Comment:
Currently, the single linting violation is because `register_step_ants_slice_regularized_registration` comes from a preexisting `*` input. 

I'm going to leave this for now, as it's out of scope of this PR, but I wanted to at least acknowledge the failing linting test.